download/testharness.go

```diff
@@ -374,6 +375,11 @@ func (t *testServer) handle(w http.ResponseWriter, r *http.Request) {
 
 	if t.expCode != 0 {
 		w.WriteHeader(t.expCode)
+
+		if t.expErrResp != "" {
```

Comment:
Since we return this body for any status code, not just errors, perhaps we should just call it `expResp` or `expBody` (since this isn't the complete response, headers and all)?

Reply:
Updated the name.